README.md

```diff
@@ -245,6 +245,7 @@ There is also an ongoing experimental project to make OCaml multiprocessor and m
   - [Modular foreign function bindings](http://openmirage.org/blog/modular-foreign-function-bindings)
   - [Py.ml](https://github.com/thierry-martinez/pyml) - OCaml bindings for Python.
 - **Editor Integration**:
+  - [ocaml-lsp](https://github.com/ocaml/ocaml-lsp) - An LSP language server for OCaml that integrates with any editor that understands LSP like [VSCode], Vim and Emacs.
```

Comment:
Did you mean to prove an URL after `[VSCode]`?

Reply:
I did, updated now.